### ISSUE TITLE
[amp story subscriptions] Tabable paywall buttons 

### ISSUE THREAD

**Issue by processprocess** [ampproject/amphtml#38110]
### Description

The following elements need to be focusable when tabing and the expected events should fire when pressing enter while they are focused.

<img width="296" alt="Screen Shot 2022-04-18 at 11 23 52 AM" src="https://user-images.githubusercontent.com/3860311/163839286-119d7a63-2368-46b1-84a7-c70605700c6f.png">

The anchor element isn't focusable because it doesn't have an `href`. 

### Alternatives Considered

n/a
